src/metabase/integrations/slack.clj

```diff
@@ -42,11 +42,15 @@
   :visibility :internal
   :type :json)
 
+(def ^:private zoned-time-epoch (t/zoned-date-time 1970 1 1 0))
+
 (defsetting slack-channels-and-usernames-last-updated
   "The updated-at time for the [[slack-cached-channels-and-usernames]] setting."
   :visibility :internal
   :cache?     false
-  :type       :timestamp)
+  :type       :timestamp
+  ;; if it has not been set, use a very early last-updated time:
```

Comment:
I think this comment is not very helpful and IMO the field kinda explains itself so I'm fine if we don't have this comment here.